specs/lib/parseGLTFSpec.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var parseGLTF = require('../../lib/parseGLTF');
+
+describe('parseGLTF', function() {
+    it('loads two shaders', function(done) {
+        var gltf = '{' +
+            '"shaders": {' +
+                '"CesiumTexturedBoxTest0FS": {' +
+                    '"type": 35632,' +
+                    '"uri": "CesiumTexturedBoxTest0FS.glsl"' +
+                '},' +
+                '"CesiumTexturedBoxTest0VS": {' +
+                    '"type": 35633,' +
+                    '"uri": "CesiumTexturedBoxTest0VS.glsl"' +
+                '}' +
+            '}' +
+        '}';
+        var frag = 'precision highp float;\n' +
```

Comment:
Instead of storing these strings, load `CesiumTexturedBoxTest0FS.glsl` or `CesiumTexturedBoxTest0VS.glsl` (no need to test both) from disk just like the test you wrote on Sunday.